examples/rayquery/shader.frag

```diff
@@ -0,0 +1,30 @@
+#version 460 core
```

Comment:
Please remove this example from the PR (and include it in the next one). It still seems incomplete and is giving me an error when I try to run it (with the input):
Cannot copy from struct of a different size (3 -> 8)!

There are already a lot of examples in this PR, so it would be nice to remove some content which seems unfinished.